smartcontractkit/external-adapters-js
# Prevent updating properties of WS connection/subscription missing fro…

## Changes

- Added checks to WS reducer to prevent updating properties of connection/subscription keys that cannot be found in state

## Steps to Test

1. Run EA with WS enabled
2. Send requests so that WS connection is opened
3. Stop sending requests, so that WS connection is closed
4. WS connection should be closed without errors

## Quality Assurance
